core/templates/dev/head/pages/exploration_editor/ExplorationStatesService.js

```diff
@@ -99,6 +99,44 @@ oppia.factory('ExplorationStatesService', [
       widget_customization_args: ['interaction', 'customizationArgs']
     };
 
+    CONTENT_ID_EXTRACTION = {
```

Comment:
These are functions -- maybe CONTENT_ID_EXTRACTORS

Reply:
Changed! (Was following the above var name `BACKEND_CONVERSIONS`!)